app/controllers/users_controller.rb

```diff
@@ -79,15 +79,14 @@ def create
       redirect_to created_user_redirect_path
     else
       unless @resource.errors[:organisation].empty?
-        @resource.errors.add(:organisation_id, message: @resource.errors[:organisation])
         @resource.errors.delete(:organisation)
```

Comment:
Still not sure why we need to do this, I know it has to do with `organisation` VS `organisation_id`

Reply:
It's so that the form builder gem links the error to the correct form field. i.e. because the form field references `organisation_id` the error needs to be on `organisation_id` rather than on `organisation`.

This PR makes it so that the model now does that anyway, but it also produces a duplicate(ish) error on `organisation` that we don't want/need to display on the view so we remove it here.